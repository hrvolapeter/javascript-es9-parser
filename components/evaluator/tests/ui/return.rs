function a() {
    return 1;
    console.log("unreachable");
    return 2;
}
console.log(a());

function b() {
    return;
}
console.log(b());

function c() {
    if (true) {
        return 1;
    }
    return 2;
}
console.log(c());
