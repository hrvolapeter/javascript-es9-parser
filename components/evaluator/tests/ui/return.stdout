1
undefined
1
